CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@
 - Fix: Removing trailing slashes for path of URI before treating the request (Issue #828)
 - Fix: Fixed a bug regarding subscriptions with empty entity-type (Issue #691)
 - Fix: updateContext propagated to the entire Service Path subtree, instead of just scoping to the particular Service Path in the Fiware-Service header (Issue # 885)
+- Mongo driver migrated to legacy-1.0.2
```

Comment:
We don't use to include such internal changes in CNR. However, I think the impact of the change could be large enough to deserve this entry. 

NTC (just informative)